package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "jempi-web",
   "version": "0.1.0",
+  "type": "module",
```

Comment:
@Lumiere-Lodi Why did you need to add this ?

Reply:
my bad. I was testing something with Matt. No worry